apps/server/src/middleware/exception-logger.ts

```diff
@@ -16,6 +16,13 @@ export class ExceptionLogger implements ExceptionFilter {
   catch(exception: Error, host: ArgumentsHost) {
     // with useGlobalFilters, the context is always HTTP
     const ctx = host.switchToHttp();
+
+    // TODO: temporary, remove this after stable
+    if (host.getType() !== 'http') {
+      this.logger.error(`Unexpected request type caught: ${host.getType()}`);
```

Comment:
provide more information here, maybe request path or something that helpful to debug